config/initializers/flood_risk_engine.rb

```diff
@@ -0,0 +1,3 @@
+FloodRiskEngine.configure do |config|
+  config.redirection_url_on_location_unchecked = "/pages/contact_ea_location"
+end
```

Comment:
I have a spearate PR #6 that also sets the default layout for the engine - can be merged later